include/zenoh-pico/collections/string.h

```diff
@@ -90,8 +90,8 @@ _z_string_t _z_string_from_bytes(const _z_bytes_t *bs);
  *   char **_val: A pointer to the array.
  */
 typedef struct {
-    size_t _len;
-    char **_val;
+    char **val;
+    size_t len;
 } _z_str_array_t;
```

Comment:
Order types in structs from bigger size to smaller size.

Reply:
Field order should be the same as in zenoh-c, to make { .val = xxx, .len = yyy } initializers work. Ok, will undo this change and do the field order change in zenoh-c